### ISSUE TITLE
Can't start debugger on Pycharm 2023.3 EAP

### ISSUE THREAD

**Issue by andyp05** [reloadware/reloadium#174]
## Describe the bug*
Start debugger

```
E:\Python\python.exe -m reloadium_launcher pydev_proxy -X pycache_prefix=C:\Users\xxxx\AppData\Local\JetBrains\PyCharm2023.3\cpython-cache "C:/Program Files/JetBrains/PyCharm/plugins/python/helpers/pydev/pydevd.py" --multiprocess --client 127.0.0.1 --port 62520 --file E:\Projects\xxxx\manage.py runserver localhost:8000 

Unexpected flag "-X"

Process finished with exit code 1
```

Regular debugger runs fine
```
E:\Python\python.exe -X pycache_prefix=C:\Users\xxxx\AppData\Local\JetBrains\PyCharm2023.3\cpython-cache "C:/Program Files/JetBrains/PyCharm/plugins/python/helpers/pydev/pydevd.py" --multiprocess --client 127.0.0.1 --port 63062 --file E:\Projects\xxxx\manage.py runserver localhost:8000
```

Uninstalled Pycharm and reloadium and tried again. No luck.

Windows 11 Pro
Pycharm 2023.3 EAP 5
Reloadium 1.3.3
Python 3.11.6

**Comment by ccolas** [comment]
Same problem here on ubuntu 20.4, pycharm 2023.2.5, python 3.9, reloadium 1.3.3

**Comment by dkrystki** [comment]
The newest version of PyCharm 2023.2.5 broke Reloadium startup. 
I'm working on a new release that fixes that. 
Until then please downgrade Pycharm to 2023.2.4

**Comment by Drezzler** [comment]
@dkrystki - Thank you so much for this tool

Eagerly waiting for the new update.

**Comment by hirosh7** [comment]
PyCharm 2023.3 is out of EAP. This is still broken. Any updated on timeframe for the fix?

PyCharm 2023.3.1 (Professional Edition)
Build #PY-233.11799.298, built on December 11, 2023

**Comment by navono** [comment]
any progress?

**Comment by dkrystki** [comment]
@hirosh7 @navono I should be able to release this week or the next one.

**Comment by dkrystki** [comment]
Fixed in Reloadium 1.3.4 and PyCharm plugin 1.3.4
